tests/testthat/_snaps/extract_replace_combine.md

```diff
@@ -0,0 +1,101 @@
+# abind errors informatively
+
+    Code
+      abind(a, b)
+    Error <simpleError>
+      all <greta_array>s must have the same dimensions except on the `along`
+      dimension
+      However, dimension 1 had varying sizes: 5 and 1
+
+---
+
+    Code
+      abind(a, c, along = 5)
+    Error <simpleError>
+      invalid 'enclos' argument of type 'character'
```

Comment:
Yeah I've got no idea. My guess would be something about `c` being a function?

Reply:
Ah it was a combination of two things

First, there was a control statement that was:

```
if (!along %in% 0:n)
```

Instead of

```
if (!(along %in% 0:n))
```

Since `along` is 5 - and !5 is FALSE, this didn't make any sense...

I also forgot to pass a vector to format_error - now fixed:

```
if (!(along %in% 0:n)) {
    msg <- cli::format_error(
      c(
        "{.arg along} must be between 0 and {n}",
        "Instead {.arg along} was {.val {along}}"
      )
    )
```